src/main/java/org/opentripplanner/transit/model/basic/WheelchairAccessibility.java

```diff
@@ -1,22 +1,7 @@
 package org.opentripplanner.transit.model.basic;
 
 public enum WheelchairAccessibility {
-  NO_INFORMATION(0),
-  POSSIBLE(1),
-  NOT_POSSIBLE(2);
-
-  public final int gtfsCode;
-
-  WheelchairAccessibility(int gtfsCode) {
-    this.gtfsCode = gtfsCode;
-  }
-
-  public static WheelchairAccessibility valueOfGtfsCode(int gtfsCode) {
-    for (WheelchairAccessibility value : values()) {
-      if (value.gtfsCode == gtfsCode) {
-        return value;
-      }
-    }
-    throw new IllegalArgumentException("Unknown GTFS WheelChairBoardingType: " + gtfsCode);
-  }
+  NO_INFORMATION,
+  POSSIBLE,
+  NOT_POSSIBLE,
 }
```

Comment:
The reason we kept the GTFS codes in the enums are:
 - It serve as documentation (minor argument)
 - We had one place to maintain the mapping.
 
 I see that you have created a mapper for mapping both ways - this opens up for asymmetrical mapping (a good thing), but it also make it easier to make mistakes. This can be fixed by making one Mapper class and let it do mapping both ways.

Reply:
There are already some cases, where the mapping should differ from the internal usage, like: 

- `PickDrop` should not allow for value -1 from GTFS data
- `Direction` contains the values 0 and 1 twice, as NeTEX has more possible values than GTFS
- Empty `Direction` is represented by null in input and output data, and with -1 in in the internal data

In the new structure (4285) both input and output mappers for GTFS should probably live in `o.o.feeds.gtfs`, should I move these already now to the graph builder package. In the future GraphQL API we probably want to use something akin to the internal values any way in the API, so the need for outbound mapping should disappear after removing the current APIs.

Reply:
> In the future GraphQL API we probably want to use something akin to the internal values any way in the API, so the need for output mapping should disappear after removing the current APIs.

This is a good point, we can leave this as is for now and merge the PR.